src/components/Draggable/Draggable.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { HTML5Backend } from 'react-dnd-html5-backend'
+import { DndProvider } from 'react-dnd'
+import { css } from '@emotion/core'
+import { useTheme } from '../../styles'
+import { FilterDraggable } from './FilterDraggable'
+import { ItemTypes } from './types/ItemTypes'
+import { DefaultDraggable } from './DefaultDraggable'
+
+export type KeyMapping = {
+  keyName: string
+  formatter?: (value: string) => string
+  ordenator?: (a: string, b: string) => number
+}
+
+export interface DraggableProps<T> {
+  name: keyof T
+  type: ItemTypes
+  origin: string
+  value: string
+  filterValues?: Array<string>
+  filterState?: Set<string>
+  onDragEnd: () => void
+  onKeyNav: (dir: 'left' | 'right' | 'up' | 'down', origin: string, key?: keyof T) => void
+  handleFilterUpdate?: (key: keyof T, filtro: Set<string>) => void
+  formatter?: (value: string) => string
+}
```

Comment:
Those props are not that easy to understand. It'll be very nice if has a doc on each prop.